incubator/blockchain-client-openliberty/README.md

```diff
@@ -1,95 +1,49 @@
-        1. Set the environment variable in the file, e.g. `MSPID=<msp_id>`
-    1. IDENTITY
-        1. From the exported wallet, open the file: `wallet/admin.json`
-        1. Copy the `name` string excluding the double quotes, e.g. `admin`
-        1. Set the environment variable in the file, e.g. `IDENTITY=<name>`
-    1. CHANNEL
-        1. Name of the channel, e.g. `mychannel`
-        1. Set the environment variable in the file, e.g. `CHANNEL=<channel name>`
-    1. CONTRACT
-        1. Name of the contract, e.g. `demoContract`
-        1. Set the environment variable in the file, e.g. `CONTRACT=<contract name>`
-1. Package the stack
-    1. `cd incubator/blockchain-client-openliberty`
-    1. `appsody stack package`
-        1. This will package the stack and add the stack to your local `dev.local` repo, e.g. `dev.local/blockchain-client-openliberty`
-1. Initialize the stack
-    1. Create a project directory, e.g. `mkdir blockchain`
-    1. Navigate to the project directory, e.g. `cd blockchain`
-    1. Initialize the stack, e.g. `appsody init dev.local/blockchain-client-openliberty` 
-1. Run the stack
-    1. Find the network name of the local fabric: `docker network ls`, e.g. `1OrgLocalFabric_network`
-    1. Find the location of your `blockchain.env` file, e.g. `/Users/tnixa/blockchain/blockchain.env`
-    1. Run the stack, e.g. `appsody run --network <network name> --docker-options "--env-file=<path to env file>"`
-    
-
-
-# Open Liberty Stack
+# Blockchain Open Liberty Stack
 
 The Open Liberty stack provides a consistent way of developing microservices based upon the [Jakarta EE](https://jakarta.ee/) and [Eclipse MicroProfile](https://microprofile.io) specifications. This stack lets you use [Maven](https://maven.apache.org) to develop applications for [Open Liberty](https://openliberty.io) runtime, that is running on OpenJDK with container-optimizations in OpenJ9.
 
 The Open Liberty stack uses a parent Maven project object model (POM) to manage dependency versions and provide required capabilities and plugins.
 
-This stack is based on OpenJDK with container-optimizations in OpenJ9 and `Open Liberty v19.0.0.12`. It provides live reloading during development by utilizing the "dev mode" capability in the liberty-maven-plugin.  To see dev mode in action (though not in Appsody) check out this [shorter demo](https://openliberty.io/blog/2019/10/22/liberty-dev-mode.html) and this  [a bit longer demo](https://blog.sebastian-daschner.com/entries/openliberty-plugin-dev-mode).
+This stack is based on OpenJDK with container-optimizations in OpenJ9 and `Open Liberty v20.0.0.3`. It provides live reloading during development by utilizing the "dev mode" capability in the liberty-maven-plugin.  To see dev mode in action (though not in Appsody) check out this [shorter demo](https://openliberty.io/blog/2019/10/22/liberty-dev-mode.html) and this  [a bit longer demo](https://blog.sebastian-daschner.com/entries/openliberty-plugin-dev-mode).
```

Comment:
I would suggest avoiding direct references to the levels of Liberty - just say, it's an extension of the "openliberty" Kabanero stack (we'll have to fix this later).

Reply:
This is exactly what is in the parent readme, so I tried to keep everything the base liberty users would expect to see.